README.md

```diff
@@ -1,58 +1,31 @@
 # IO Functions template
```

Comment:
Change the title on the readme.

Reply:
Done in [aa5607c](https://github.com/pagopa/io-functions-lollipop/pull/19/commits/aa5607cbdc1c198e1b2a8d004c7eb90ff18cfe6d)